docs/_source/components-overview/pop_syn/population_params.rst

```diff
@@ -235,50 +392,117 @@ Multiple prescriptions are implemented and can be selected.
 If a standard prescription is used, interpolators are available to speed up the calculation and predict additional properties of the collapse, such as the spin.
 
 
-.. list-table::
-   :widths: 50 50
-   :header-rows: 1
+Step Supernova
+~~~~~~~~~~~~~~
 
-   * - Parameter
-     - Description/Value
-   * - `import`
-     - ['posydon.binary_evol.SN.step_SN', 'StepSN']
-   * - `absolute_import`
-     - None ('package' kwarg for importlib.import_module)
-   * - `mechanism`
-     - 'Patton&Sukhbold20-engine' (options are: 'direct', Fryer+12-rapid', 'Fryer+12-delayed', 'Sukhbold+16-engine', 'Patton&Sukhbold20-engine')
-   * - `engine`
-     - 'N20' (options are 'N20' for 'Sukhbold+16-engine', 'Patton&Sukhbold20-engine' or None for the others)
-   * - `PISN`
-     - 'Marchant+19' (options are None, "Marchant+19")
-   * - `ECSN`
-     - "Podsiadlowksi+04" (options are "Tauris+15", "Podsiadlowksi+04")
-   * - `conserve_hydrogen_envelope`
-     - True
-   * - `max_neutrino_mass_loss`
-     - 0.5 (float in [0,inf])
-   * - `max_NS_mass`
-     - 2.5 (float in [0,inf])
-   * - `use_interp_values`
-     - True (if True, use interpolation values for the SN properties, which are calculated during ``step_MESA``)
-   * - `use_profiles`
-     - True
-   * - `use_core_masses`
-     - True
-   * - `approx_at_he_depletion`
-     - False
-   * - `kick`
-     - True
-   * - `kick_normalisation`
-     - 'one_over_mass' (options are "one_minus_fallback", "one_over_mass", "NS_one_minus_fallback_BH_one", "one", "zero")
-   * - `sigma_kick_CCSN_NS`
-     - 265.0 (float in [0,inf])
-   * - `sigma_kick_CCSN_BH`
-     - 265.0 (float in [0,inf])
-   * - `sigma_kick_ECSN`
-     - 20.0 (float in [0,inf])
-   * - `verbose`
-     - False
+This step performs the core-collapse supernova evolution of the star.
+Multiple prescriptions are implemented and can be selected.
+If a standard prescription is used, interpolators are available to speed up the calculation and predict additional properties of the collapse, such as the spin.
+
+.. list-table::
+  :widths: 10 80 10
+  :class: population-params-table
+  :header-rows: 1
+
+  * - Parameter
+    - Description
+    - Default Value
+
+  * - ``import``
+    - | The import path for the step and the name of the step class.
+    - ``['posydon.binary_evol.SN.step_SN', 'StepSN']``
+
+  * - ``absolute_import``
+    - | An absolute import of a custom step. It follows the same structure as ``import``.
+      | ``['import.path', 'name_of_step']``
+    - ``None``
+
+  * - ``mechanism``
+    - | The mechanism used for the supernova.
+     
+      * ``'direct'``
+      * ``'Fryer+12-rapid'``
+      * ``'Fryer+12-delayed'``
+      * ``'Sukhbold+16-engine'``
+      * ``'Patton&Sukhbold20-engine'``
+    - ``'Patton&Sukhbold20-engine'``
+
+  * - ``engine``
+    - | The engine used for the supernova.
+      | Relevant for ``'Sukhbold+16-engine'`` and ``'Patton&Sukhbold20-engine'`` mechanisms.
+    - ``'N20'``
```

Comment:
For those SN parameters, we are bound to the defined combinations in [MODELS.py](https://github.com/POSYDON-code/POSYDON/blob/max_tutorial_updates/posydon/grids/MODELS.py).
On the engine I have the experience, leaving it to N20, while using direct on mechanism will make the code fail. By the way, it looks like we have tables for other engines of `Sukhbold+16`, shouldn't we tell users about them?

Reply:
That's not fully true. If you turn off `use_interp_values`, you should be able to use whatever combination you want.

I'm not familiar with the other engines that are implemented. If they're working, we can add them.

Reply:
If you switch `use_interp_values` off you request profiles to exist. (That's what we do in the post-processing) Hence, it should fail either because the MODEL is missing or the profiles are missing.

I don't know about the other engines either, I have only seen they are there, whether they are working or not you'd need to ask the person, who implemented them (I'd guess @ssbvr ).

Reply:
It's more complex than that, since it also depends on the `use_core_masses`. I've added this to reflect this nuance.